change/react-native-windows-2020-04-24-19-55-04-master.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
prerelease is still the right one for 0.60. (This is not at all confusing 🙂)

Basically, for the 0.61 release we moved to a model where we follow semver and update the patch number if we make fixes, but don't make breaking changes within a released version. Future stable branches will look like 0.61, but 0.60 is weird in that all of its releases end with the "vnext" prerelease tag.